dev/ci/ci-quickchick.sh

```diff
@@ -11,6 +11,6 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/quickchick"
   make
-  make install INSTALLDIR=$CI_INSTALL_DIR/bin
+  make install
```

Comment:
this will make quickchick install in some global path when run locally

Reply:
I believe I fixed the problem upstream by using the path of coqc rather than the bin folder of opam as the default

Reply:
oh in the overlay so not yet upstream
Are quickchick devs ok with that?